### PR TITLE
Update replicatedhq/troubleshoot to v0.62.1

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*
This PR updates the component replicatedhq/troubleshoot to v0.62.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by taneyland** [comment]
/hold

holding for manually test with support bundle

File: projects/replicatedhq/troubleshoot/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The EKS-A diagnostic bundle functionality is built on top of troubleshoot.
 1. Compare the old tag to the new, looking specifically for Makefile changes.
```
Comment (abhay-krishna) [inline projects/replicatedhq/troubleshoot/README.md:18]: Let's update the version badge in README as well

File: projects/replicatedhq/troubleshoot/README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The EKS-A diagnostic bundle functionality is built on top of troubleshoot.
    - The make target `make binaries` will generate linux and darwin binaries for Troubleshoot, outputting them to `./_output/bin/troubleshoot`
    - For more options, see [the Development documentation](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/building-locally.md)
 1. add the `support-bundle` binary for your platform to your path
-1. set the env var `MR_TOOLS_DISABLE=false`, so that the EKS-A commands will use local binaries
+1. set the env var `MR_TOOLS_DISABLE=true`, so that the EKS-A commands will use local binaries
```
Comment (abhay-krishna) [inline projects/replicatedhq/troubleshoot/README.md:36]: Nice catch!

**Review by abhay-krishna — APPROVED** [review]

**Pull request merged**